deepfence_agent/tools/apache/scope/probe/kubernetes/reporter.go

```diff
@@ -346,8 +358,41 @@ func (r *Tagger) Tag(rpt report.Report) (report.Report, error) {
 	return rpt, nil
 }
 
-// Report generates a Report containing Container and ContainerImage topologies
+func (r *Reporter) updateReportCache() {
+	k8sReport, err := r.generateReport()
+	if err != nil {
+		logrus.Error(err.Error())
```

Comment:
If we get an error, shouldn't we return ? Or do something else ?

Reply:
This is first report collection, on start up. If it is a temporary error, it will be resolved in the next generated report (after 5 minutes)